pkg/asset/internal/templates.go

```diff
@@ -156,6 +156,33 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 `)
 
+	KubeletApproverTemplate = []byte(`apiVersion: extensions/v1beta1
```

Comment:
Can we add a comment to the top of this with a note/warning that this should not be used in production clusters (maybe say because it will auto-approve all CSRs)